x-pack/test/functional_endpoint/apps/endpoint/resolver.ts

```diff
@@ -0,0 +1,281 @@
+      for (let i = 0; i < NewNodeDataStyle.length; i++) {
+        expect(parseFloat(NewNodeDataStyle[i].left)).to.lessThan(
+          parseFloat(OriginalNodeDataStyle[i].left)
+        );
+        expect(parseFloat(NewNodeDataStyle[i].top)).to.lessThan(
+          parseFloat(OriginalNodeDataStyle[i].top)
+        );
+      }
+      await (await testSubjects.find('center-button')).click();
+
+      const CenterNodeData = await pageObjects.endpointAlerts.getEndpointAlertResolverNodeData(
+        'resolverNode',
+        'style'
+      );
+      const CenterNodeDataStyle = await pageObjects.endpointAlerts.parseStyles(CenterNodeData);
+
+      for (let i = 0; i < CenterNodeDataStyle.length; i++) {
+        expect(parseFloat(CenterNodeDataStyle[i].left)).to.equal(
+          parseFloat(OriginalNodeDataStyle[i].left)
+        );
+        expect(parseFloat(CenterNodeDataStyle[i].top)).to.equal(
+          parseFloat(OriginalNodeDataStyle[i].top)
+        );
+      }
+    });
+
+    it('resolver Nodes navigation zoom in', async () => {
+      const OriginalNodeData = await pageObjects.endpointAlerts.getEndpointAlertResolverNodeData(
+        'resolverNode',
+        'style'
+      );
+      const OriginalNodeDataStyle = await pageObjects.endpointAlerts.parseStyles(OriginalNodeData);
+
+      await (await testSubjects.find('zoom-in')).click();
+      await browser.setWindowSize(2400, 1800);
+      const NewNodeData = await pageObjects.endpointAlerts.getEndpointAlertResolverNodeData(
+        'resolverNode',
+        'style'
+      );
+      const NewNodeDataStyle = await pageObjects.endpointAlerts.parseStyles(NewNodeData);
+
+      for (let i = 1; i < NewNodeDataStyle.length; i++) {
+        expect(parseFloat(OriginalNodeDataStyle[i].left)).to.lessThan(
+          parseFloat(NewNodeDataStyle[i].left)
+        );
+        expect(parseFloat(OriginalNodeDataStyle[i].top)).to.lessThan(
+          parseFloat(NewNodeDataStyle[i].top)
+        );
+        expect(parseFloat(OriginalNodeDataStyle[i].width)).to.lessThan(
+          parseFloat(NewNodeDataStyle[i].width)
+        );
+        expect(parseFloat(OriginalNodeDataStyle[i].height)).to.lessThan(
+          parseFloat(NewNodeDataStyle[i].height)
+        );
+        await (await testSubjects.find('zoom-out')).click();
+      }
+    });
+
+    it('resolver Nodes navigation zoom out', async () => {
+      const OriginalNodeData = await pageObjects.endpointAlerts.getEndpointAlertResolverNodeData(
+        'resolverNode',
+        'style'
+      );
+      const OriginalNodeDataStyle = await pageObjects.endpointAlerts.parseStyles(OriginalNodeData);
+      await (await testSubjects.find('zoom-out')).click();
+      const NewNodeData = await pageObjects.endpointAlerts.getEndpointAlertResolverNodeData(
+        'resolverNode',
+        'style'
+      );
+      const NewNodeDataStyle1 = await pageObjects.endpointAlerts.parseStyles(NewNodeData);
```

Comment:
why is it NewNodeDataStyle1 here, with the number added on?  its a little harder to read thru with this NewNodeDataStyle and NewNodeDataStyle1 and NewNodeDataStyle[i] - is there any disambiguation of variable names / usage you could put in to help?

Reply:
Done